commentHandler.go

```diff
@@ -12,8 +13,17 @@ import (
 	"github.com/google/go-github/github"
 )
 
-const open = "open"
-const closed = "closed"
+const openConstant string = "open"
```

Comment:
Nit picking.. what is the suffix adding?

Reply:
@rgee0

Reply:
I responded by email so it ended up below - a GitHub quirk.

You asked for the suffix to avoid keyword clashing.  Applied to all the constants for consistency.